Sources/Helpers/OAResourcesUIHelper.mm

```diff
@@ -56,6 +57,12 @@ - (void) updateSize
     // override
 }
 
+- (BOOL) isInstalled
```

Comment:
You can check if the resource is installed via ```OsmAnd::ResourcesManager::ResourceOrigin::Installed```, so this method is probably incorrect and unnecessary.